docs/UserGuide.md

```diff
@@ -155,6 +157,7 @@ Format: `edit INDEX [d/DESCRIPTION] [p/PRIORITY] [q/QUANTITY] [e/EXPIRY DATE] [t
 * Existing values will be updated to the input values.
 * When editing tags, the existing tags of the food item will be removed i.e adding of tags is not cumulative.
 * You can remove all the tags of a food item by typing `t/` without specifying any tags after it.
+* Similar to [`add`](#adding-a-food-item-add) command, a food item with the same description, expriy date and tags as another food item is considered a duplicate.
```

Comment:
Should the link be #adding-a-food-item--add instead?

Reply:
Alright updated. Thanks